tools~/run-regression-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# This script requires a running local SpacetimeDB instance.
+
+set -ueo pipefail
+
+STDB_PATH="$1"
+SDK_PATH="$(dirname "$0")/.."
+SDK_PATH="$(realpath "$SDK_PATH")"
+
+cargo run --manifest-path "$STDB_PATH/crates/cli/Cargo.toml" -- generate -y -l csharp -o "$SDK_PATH/examples~/regression-tests/client/module_bindings" --project-path "$SDK_PATH/examples~/regression-tests/server"
```

Comment:
should this be calling `gen-regression-tests.sh`?

Reply:
I had been not doing that for some reason but now I think it's fine